pkg/generators/models/testdata/cases/validation/api.yaml

```diff
@@ -11,7 +11,15 @@ components:
         - name
         - gender
         - favoriteColors
+        - cron
       properties:
+        cron:
+          type: string
+          pattern: (@(annually|yearly|monthly|weekly|daily|hourly|reboot))|(@every (\d+(ns|us|µs|ms|s|m|h))+)|((((\d+,)+\d+|(\d+(\/|-)\d+)|\d+|\*) ?){5,7})
+          x-pattern-error: "must be a valid cron value"
```

Comment:
added two new fields, one with and one without the `x-pattern-error` so that we can test the error message is produced correctly